distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Filter_Condition_Helpers.enso

```diff
@@ -78,6 +78,7 @@ make_filter_column source_column:Column filter_condition on_problems:Problem_Beh
         Is_In values _ ->
             warn_on_nothing_in_comparison_vector filter_condition values <|
                 source_column.is_in values
+        Predicate p -> Column.from_vector source_column.name (source_column.to_vector.filter p)
```

Comment:
This is not really correct, as it changes type of `Column` (e.g. `DB_Column` to memory one) as well as type of the column itself. However I cannot find `filter` method on the `Column`.